src/pages/workspaces/workspace/analysis/runtime-common.js

```diff
@@ -150,16 +150,9 @@ export const SaveFilesHelpRStudio = () => {
 export const SaveFilesHelpGalaxy = () => {
   return h(Fragment, [
     p([
-      'Deleting your Cloud Environment will stop your ',
```

Comment:
this seems to be the msg for galaxy app deletion?

Reply:
I can put some of this back in, I just took it out because there was very similar messaging [here](https://github.com/DataBiosphere/terra-ui/pull/3636/files#:~:text=%27Deleting%20your%20cloud%20environment%20will%20stop%20all%20running%20notebooks%20and%20associated%20costs.%20You%20can%20recreate%20your%20cloud%20environment%20later%2C%20%27%2C) already

Reply:
can u share some screen shots what the msg will look like?

Galaxy app isn't the same as notebooks...so `notebooks` shouldn't be mentioned when user is deleting galaxy app

Reply:
<img width="1481" alt="Screen Shot 2023-01-06 at 10 20 11 AM" src="https://user-images.githubusercontent.com/13465644/211043127-9a734f9d-32cf-48e6-bf83-f65ae9287b60.png">
FYI this is what I am seeing when deleting a GCP Galaxy app

Reply:
updated to put in a new link and add back galaxy specific messaging